### PR TITLE
feat(refactor): ✨ remove brew command and add proper ollama installation for colab to run in free tier

### LINKED ISSUE #123: Can "Building A RAG Ebook "Librarian" Using LlamaIndex" be run using Google Colab
I'm wondering whether [Building A RAG Ebook "Librarian" Using LlamaIndex](https://huggingface.co/learn/cookbook/rag_llamaindex_librarian) could be run using Google Colab in its current format (without Pro account).

If not, maybe it would be a good idea to add that idea explicitly or maybe include installation instructions for Google Colab Pro (where you can access the terminal window).

I have the installation instructions for Ollama in Google Colab Pro here for reference: https://github.com/sergiopaniego/RAG_local_tutorial?tab=readme-ov-file#intructions-to-run-the-example-using-google-colab-pro-account-needed

I know that the notebook says "locally" but the [Open in Colab button](https://huggingface.co/learn/cookbook/rag_llamaindex_librarian) may be misleading.

@jinnovation

### PR DESCRIPTION
**Pull request by onuralpszr**
# What does this PR do?

This PR refactors the setup for Ollama in Colab by removing the Homebrew command and adding a compatible installation method for the google colab free tier without requirement of "terminal/console"

Fixes #123

## Who can review?

@merveenoyan and @stevhliu.

### CODE REVIEW

File: notebooks/en/rag_llamaindex_librarian.ipynb [hunk 1]
```diff
@@ -72,13 +72,74 @@
     "    llama-index-llms-ollama"
```
Comment (stevhliu) [inline notebooks/en/rag_llamaindex_librarian.ipynb:72]: <p>Ollama installation</p><p> </p><p>These dependencies help properly detect the GPU.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (stevhliu) [inline notebooks/en/rag_llamaindex_librarian.ipynb:72]: <p>Install Ollama.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (stevhliu) [inline notebooks/en/rag_llamaindex_librarian.ipynb:72]: <p>"Run Ollama service in the background."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (stevhliu) [inline notebooks/en/rag_llamaindex_librarian.ipynb:72]: <p>Pull Llama2 from the Ollama library.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Review by stevhliu — APPROVED** [review]
Added a few notes but LGTM overall, thanks!

**Comment by onuralpszr** [comment]
@stevhliu thank you and all changes are done.

**Comment by merveenoyan** [comment]
the CI times out, there's no issue regarding this notebook.

**Comment by onuralpszr** [comment]
> the CI times out after 6 hrs, there's no issue regarding this notebook.

Yup :)

**Comment by onuralpszr** [comment]
Hello, I was waiting for a while and Is there anything I should do or check about it ? 

Thank you.

cc @sergiopaniego

**Comment by sergiopaniego** [comment]
> Hello, I was waiting for a while and Is there anything I should do or check about it ?
> 
> Thank you.
> 
> cc @sergiopaniego

Thanks for solving the issue I created!! 😄 
I think there is nothing else to be done, it'll be merged soon. Let's be patient 😃

**Review by sergiopaniego — APPROVED** [review]

**Comment by onuralpszr** [comment]
@stevhliu  I did merge but still looks stuck on CI I was checking what causing it.  I also notice other comment on other PR so I wanted to let you know.

**Comment by stevhliu** [comment]
How odd! Can you try getting the latest changes from `main` again to see if that helps? It seemed to have worked for #238, but I see that it is stalling out again in several other PRs.

**Comment by onuralpszr** [comment]
@stevhliu  I did already If you check my merge commit, I will do it again but that's for new cookbook so won't make big difference anyway

**Comment by stevhliu** [comment]
Ok, with the latest fix in #239 the CI should pass now. Can you update and get the latest fix again? Thanks again for your patience! 🤗

**Comment by onuralpszr** [comment]
@stevhliu well it is merged but I cannot see URL ? :) at least I wish to see dev-docs :)

**Comment by stevhliu** [comment]
You can see them [here](https://huggingface.co/learn/cookbook/rag_llamaindex_librarian#ollama-installation)

**Pull request merged**